strawberryfields/cli/__init__.py

```diff
@@ -0,0 +1,231 @@
+
+    Args:
+        args (ArgumentParser): arguments that were specified on the command
+            line stored as attributes in an argument parser object
+    """
+    if args.token:
+        kwargs = {"authentication_token": args.token}
+    else:
+        kwargs = configuration_wizard()
+
+    if args.local:
+        store_account(**kwargs, location="local")
+    else:
+        store_account(**kwargs)
+
+def ping():
+    """Tests the connection to the remote backend."""
+    if Connection().ping():
+        sys.stdout.write("You have successfully authenticated to the platform!\n")
+    else:
+        sys.stdout.write("There was a problem when authenticating to the platform!\n")
+
+def configuration_wizard():
+    r"""Provides an interactive selection wizard on the command line to
+    configure every option for the API connection.
+
+    Default configuration options are provided as defaults to the user.
+    Configuration options as detailed in :doc:`/introduction/configuration`.
+
+    Returns:
+        dict[str, Union[str, bool, int]]: the configuration options
+    """
+    default_config = create_config()["api"]
+
+    # Getting default values that can be used for as messages when getting inputs
+    hostname_default = default_config["hostname"]
+    ssl_default = "y" if default_config["use_ssl"] else "n"
+    port_default = default_config["port"]
+
+    authentication_token = (
+        input("Please enter the authentication token to use when connecting: [] ")
+    )
+
+    if not authentication_token:
+        sys.stdout.write("No authentication token was provided, please configure again.")
+        sys.exit()
+
+    hostname = (
+        input("Please enter the hostname of the server to connect to: [{}] ".format(hostname_default)) or hostname_default
+    )
+
+    ssl_input = input("Should the client attempt to connect over SSL? [{}] ".format(ssl_default)) or ssl_default
+    use_ssl = ssl_input.upper() == "Y"
+
+    port = input("Please enter the port number to connect with: [{}] ".format(port_default)) or port_default
+
+    kwargs = {
+        "authentication_token": authentication_token,
+        "hostname": hostname,
+        "use_ssl": use_ssl,
+        "port": port,
+    }
+    return kwargs
+
+def run_blackbird_script(args):
+    """Run a blackbird script.
+
+    Related arguments:
+    * input: the input blackbird script to be run
+    * output: the output file to store the results in (optional)
+
+    Args:
+        args (ArgumentParser): arguments that were specified on the command
+            line stored as attributes in an argument parser object
+    """
+    try:
+        program = load(args.input)
+    except FileNotFoundError:
+        sys.stdout.write("The {} blackbird script was not found.".format(args.input))
+        sys.exit()
+
+    eng = RemoteEngine(program.target)
+
+    sys.stdout.write("Executing program on remote hardware...\n")
+    result = eng.run(program)
```

Comment:
Ah, I noticed something with the `RemoteEngine` implementation that may not be ideal... The `run()` method accepts a `shots` argument, which overrides the value specified in the Blackbird script (which it probably shouldn't), and defaults to 1 if not given. So this line here would run every program with 1 shot regardless of what is in the script, which is probably not the intended behaviour. My bad for not catching this earlier!

I think the best thing to do for now might be to explicitly pass in `shots` here, i.e.

```python
result = eng.run(program, shots=program.run_options["shots"])
```
but ideally, I think this line should be eventually modified to only override `shots` if it is explicitly passed in:

https://github.com/XanaduAI/strawberryfields/blob/447983a03c09a6a44f982f7a92fdd06f2f147f4c/strawberryfields/api/connection.py#L151

Reply:
Thanks @pauljxtan for catching this! Do you want to create a quick PR to master to fix this?

Reply:
Sure thing, I will get that done ASAP!